MIGRATION.md

```diff
@@ -179,7 +179,6 @@ Additionally, multiple components and directives are now standalone components,
 - ComponentLoaderDirective
 - ThemeColorDirective
 - SlideButtonComponent
-- PageLocalNavigationComponent
```

Comment:
This should probably not be removed from the migration-notes for v8 😊

Reply:
Reverted 👍🏻